From 6b38748d8943595f98b86977c5b62335edf33b44 Mon Sep 17 00:00:00 2001
From: karthikeyana <karthikeyan.a@live.com>
Date: Sun, 20 Aug 2017 03:11:41 +0530
Subject: [PATCH] Tagging features added

---
 src/gitLifeCycle.sh | 14 ++++++++++++++
 1 file changed, 14 insertions(+)

diff --git a/src/gitLifeCycle.sh b/src/gitLifeCycle.sh
index 3f1144f..da2a429 100644
--- a/src/gitLifeCycle.sh
+++ b/src/gitLifeCycle.sh
@@ -147,3 +147,17 @@ git reset --hard HEAD~
 git status -s
 git reset --hard acee1e6c32c86d308ac0f6b7f3ce7c54af46eb26
 git status -s
+
+13) TAG OPERATION
+create tag:
+git tag -a 'version-1.0' -m 'Tagged basic description of git' HEAD
+git push origin tag version-1.0
+
+view tag:
+git tag -l
+git show version-1.0
+
+delete tag:
+git tag -d version-1.0
+git push origin :version-1.0
+
-- 
2.7.4

